src/easystar.js

```diff
@@ -362,27 +374,29 @@ EasyStar.js = function() {
                 path.reverse();
                 var ip = path;
                 instance.callback(ip);
-                return
```

Comment:
Nice. This looks like it was actually a bug. The calculation should have continued for other instances here. Nice find.